src/event-input/calcPb.js

```diff
@@ -43,9 +45,8 @@ const calcPb = (result, runner) => {
     let bestAgeGrade = runner.stats.recordsAgeGrade.best;
     let worstAgeGrade = runner.stats.recordsAgeGrade.worst;
 
-    if (!bestAgeGrade) {
+    if (!bestAgeGrade /* Therefore also !worstAgeGrade */) {
```

Comment:
Same as the previous comment about your comment, perhaps you could explain a bit better here. It almost seems like you're saying if(!bestAgeGrade) === if (!worstAgeGrade) which may also be true, but let's remove ambiguity